### PR TITLE
Update build config to use dev app and key

### PR DESCRIPTION
**Pull request by roundhill**
This PR adds a hard coded dev app key to the app so that developers will not need to create a Simperium account and app in order to build and test the app.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -20,18 +16,13 @@ Sample `local.properties`
...
-simperiumAppKey=SIMPERIUM_KEY
-googleAnalyticsId=
-```
-
 * Install debug build with Android Studio or:
 ```shell
 ./gradlew installDebug
 ```
 
+Sign up for a new account within the app. Use the account for **testing purposes only** as all note data will be periodically cleared out on the server.
```
Comment (nbradbury) [inline README.md]: Should there be a note here **not** to sign in using an existing account?

Comment (roundhill) [inline README.md]: Thanks for the copy idea in Slack, updated in 93d9e43

**Comment by nbradbury** [comment]
Looks good! :shipit:

**Comment by roundhill** [comment]
Thanks for the review @nbradbury!

**Pull request merged**